.travis.yml

```diff
@@ -2,16 +2,29 @@ language: cpp
 
 matrix:
   include:
-    - os: linux
+    - name: 'openssl'
+      os: linux
       compiler: clang
-
-before_script:
-  - source ./.travis/install_cmake.sh
-  - source ./.travis/install_openssl.sh
-
-script:
-  - install -d build
-  - cd build
-  - cmake -DCMAKE_CXX_COMPILER=$CXX -DCMAKE_C_COMPILER=$CC ..
-  - make VERBOSE=1
-  - ctest
+      before_script:
+        - source ./.travis/install_cmake.sh
+        - source ./.travis/install_openssl.sh
+      script:
+        - install -d build
+        - cd build
+        - cmake -DCMAKE_CXX_COMPILER=$CXX -DCMAKE_C_COMPILER=$CC ..
+        - make VERBOSE=1
+        - ctest
+    - name: 'boringssl'
+      os: linux
+      compiler: clang
+      before_script:
+        - source ./.travis/install_cmake.sh
+        - source ./.travis/install_go.sh
```

Comment:
@twifkak let me know if there's a preferred way to install the golang dependency

Reply:
Nope, that works for me. I guess your previous commit using the Travis `go: 1.13` directive didn't work?

Reply:
ya. i think the `go:` directive is just for go environments. docs were sparse, though ...